test/gtest/group_conv.hpp

```diff
@@ -548,3 +560,25 @@ std::vector<miopenTensorLayout_t> GetLayoutValues()
     DEFINE_GROUP_CONV_TEST(2, alpha, beta, type, dir, ab_case)
 #define DEFINE_GROUP_CONV3D_TEST(type, dir, alpha, beta, ab_case) \
     DEFINE_GROUP_CONV_TEST(3, alpha, beta, type, dir, ab_case)
+
+//// 2D
+// Bilinear where alpha = 2.2 and beta 3.3
+#define DEFINE_GROUP_CONV2D_BILINEAR_TEST(type, dir) \
```

Comment:
Don't like these new macros split by case. Define some values for `alpha` and `beta` (2 each) separately on lines 555 and 556 and don't define any new macros.